### DISCUSSION TITLE
v1.3.3

### DISCUSSION THREAD

**Discussion by buger** [probelabs/goreplay#1016] [General]
## What's Changed
- Various improvements for packet capture, especially for POST requests https://github.com/buger/goreplay/pull/1007
- Fixed replay of pcap files https://github.com/buger/goreplay/pull/1007
- Fixed replay of files with malformed records https://github.com/buger/goreplay/pull/1015

**Full Changelog**: https://github.com/buger/goreplay/compare/v1.3.2...1.3.3

<hr /><em>This discussion was created from the release <a href='https://github.com/buger/goreplay/releases/tag/1.3.3'>v1.3.3</a>.</em>

**Comment by yogesh23may** [comment]
[DEBUG][elapsed 761.2µs]: [INPUT-FILE] FileInput: end of file '/scripts/go-replay-deploy-5dfbcf5cf-bvzwq_30_11_2021-06_52_0.gor'

Records found: 3
Malformed records: <nil>
Files processed: 1
Bytes processed: 1478
Max wait: 4.821309157s
Min wait: 4.216318997s
First wait: 4.821309157s
It will take `9.037628154s` to replay at current speed.
Found 0 records with out of order timestamp

This is from the log , but no GET request is received on the actual server .
